examples/customize-toolbar-excel-download/index.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import MUIDataTable from '../../src';
+import CustomToolbar from './CustomToolbar';
+import XLSX from 'xlsx';
```

Comment:
This import presents a problem for those looking to run this locally, as we won't have access to this package without adding it manually. Ideally, the examples would run without having to download external dependencies. Is there a way to perhaps mock this package locally so that it will run and demonstrate the code works without necessarily needing all the excel transformations? Maybe this example is too ambitious.

Reply:
It’s not really doing much with xlsx except converting a 2d array to a sheet and downloading it. Maybe just comment out that stuff and the import? That way if someone wants to use it as a template they have the code.

Reply:
The problem is that the maintainers of this library will be responsible for anything put into it, so if it's unclear how a particular example works, it could lead to people opening issues with questions about it, that then require attention. Alternatively, if the XLSX code is included in the example, it's a lot of extra code in this project for a single example.

I think it would be best at this point to just remove the XLSX-dependent code so that it doesn't lead to confusion or additional questions about how to make it work. Excel downloads in general were something that this library has steered away from providing because it's a lot of additional code, and there's a reasonable desire to keep things a bit trim, instead of becoming a humongous, everything-to-everyone style of framework.

At some point, it may be good to add the ability to handle 3rd-party plugins to extend the functionality of the library without bloating it, and at that point, we can reconsider more complex examples and use cases.

Reply:
Sounds good to me. I'll pull it out.

Reply:
I added this PR to demonstrate what I meant: https://github.com/gregnb/mui-datatables/pull/651. No need to pull everything, just the parts specific to the XLSX package. You can add the alterations here or we can move forward with the new PR, it's up to you.

Reply:
I just renamed the example folder name since we're not doing an excel download anymore.